core/src/components/input/input.ionic.outline.scss

```diff
@@ -0,0 +1,162 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 4px;
+  --padding-start: 12px;
+  --padding-end: 12px;
+
+  min-height: 40px;
+}
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.input-fill-outline) .input-bottom {
+  border-top: none;
+}
+
+/**
+ * Outline inputs do not have a bottom border.
+ * Instead, they have a border that wraps the
+ * input + label.
+ */
+:host(.input-fill-outline) .input-wrapper {
+  border-bottom: none;
+}
+
+:host(.input-fill-outline.input-label-placement-stacked) .label-text-wrapper,
+:host(.input-fill-outline.input-label-placement-floating) .label-text-wrapper {
+  @include transform-origin(start, top);
+
+  position: absolute;
+
+  /**
+   * Label text should not extend
+   * beyond the bounds of the input.
+   */
+  max-width: calc(100% - var(--padding-start) - var(--padding-end));
+}
+
+/**
+ * The label should appear on top of an outline
+ * container that overlaps it so it is always clickable.
+ */
+:host(.input-fill-outline) .label-text-wrapper {
+  position: relative;
```

Comment:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29268/commits/038f2f99cd5ae74b71fd5262b127947555583ff2. This necessitated a bit of rework because the label text was previously being translated above the `<label>` element (and the whole component, for that matter), causing clicks to not be captured. I had to rearrange things so that the `<label>` bounds cover both the label text and the native input wrapper, which in turn required moving the outline since otherwise it would surround the label text as well.

Reply:
Works!